source/octf/proto/defs.proto

```diff
@@ -87,3 +87,16 @@ enum OutputFormat {
         (opts_enumval).cli_switch = "csv"
     ];
 }
+
+enum CacheType {
```

Comment:
Here we expect only very generic types, so let's move it the interface file definition respectively.